api/v1alpha1/dbaasinstance_conversion.go

```diff
@@ -66,3 +58,42 @@ func (dst *DBaaSInstance) ConvertFrom(srcRaw conversion.Hub) error {
 
 	return nil
 }
+
+// ConvertTo convert this DBaaSInstanceSpec to v1beta1.DBaaSInstanceSpec
+func (src *DBaaSInstanceSpec) ConvertTo(dst *v1beta1.DBaaSInstanceSpec) {
+	dst.InventoryRef = v1beta1.NamespacedName(src.InventoryRef)
+	dst.ProvisioningParameters = map[v1beta1.ProvisioningParameterType]string{}
+	dst.ProvisioningParameters[v1beta1.ProvisioningName] = src.Name
+	dst.ProvisioningParameters[v1beta1.ProvisioningCloudProvider] = src.CloudProvider
+	dst.ProvisioningParameters[v1beta1.ProvisioningRegions] = src.CloudRegion
+	for key, val := range src.OtherInstanceParams {
+		switch key {
+		case "project":
+			// For MongoDB Atlas
+			dst.ProvisioningParameters[v1beta1.ProvisioningTeamProject] = val
+		case "engine":
```

Comment:
```suggestion
		case "Engine":
```
It should be `Engine`: https://github.com/RHEcosystemAppEng/rds-dbaas-operator/blob/19e0d6bfb985cfb82e35ecd9b4ac94d6b1d1aa4f/controllers/rdsinstance_controller.go#L55

Reply:
Corrected.